mlreco/iotools/parsers/__init__.py

```diff
@@ -118,8 +118,9 @@
     parse_cluster3d_full,
     parse_cluster3d_types,
     parse_cluster3d_kinematics,
-    parse_cluster3d_kinematics_full_clean,
     parse_cluster3d_kinematics_clean,
+    parse_cluster3d_clean_full_extended,
+    parse_cluster3d_full_extended,
```

Comment:
Can you clarify what these are for? Do we need to use them instead of `parse_cluster3d_clean_full` to run GraphSpice now?

Reply:
These were parsers for vertex net, so not really related to graphspice. This one includes all variables in `parse_cluster3d_full` with the addition of vertex labels.

Reply:
Alright, might want to rename them at some point but that works for now.